Project-Aurora/Project-Aurora/Settings/Control_Settings.xaml

```diff
@@ -277,8 +277,9 @@
                 <Grid>
                     <Button x:Name="updates_check" Content="Check for Updates" HorizontalAlignment="Left" Margin="10,31,0,0" VerticalAlignment="Top" Click="updates_check_Click"/>
                     <CheckBox x:Name="updates_autocheck_on_start" Content="Automatically check for updates on program start" HorizontalAlignment="Left" Margin="10,56,0,0" VerticalAlignment="Top" Checked="updates_autocheck_on_start_Checked" Unchecked="updates_autocheck_on_start_Checked"/>
+                    <CheckBox x:Name="pointer_updates_autocheck_on_start" Content="Automatically check for pointer updates on program start" HorizontalAlignment="Left" Margin="10,76,0,0" VerticalAlignment="Top" Checked="pointer_updates_autocheck_on_start_Checked" Unchecked="pointer_updates_autocheck_on_start_Checked"/>

```

Comment:
This should be binding to the Property in Configuration, not using event handlers, see L169